addOns/graphql/src/main/resources/org/zaproxy/addon/graphql/resources/Messages.properties

```diff
@@ -33,3 +33,17 @@ graphql.error.importfile = An error occurred while importing from file.
 graphql.error.filenotfound = Cannot find the specified file.
 graphql.error.emptyendurl = Endpoint URL must be specified.
 graphql.error.invalidurl = Please enter a valid URL.\n{0}
+
+graphql.options.panelName = GraphQL Options
+graphql.options.label.queryDepth = Maximum Query Depth
+graphql.options.label.argsDepth = Maximum Arguments Depth
+graphql.options.label.split = Generate Query For
+graphql.options.label.requestMethod = Request Method
+graphql.options.label.optionalArgsEnabled = Specify Optional Arguments
```

Comment:
I think these should have trailing colons `:` to be more consistent with other options panels.